Project.toml

```diff
@@ -4,20 +4,23 @@ version = "0.6.9"
 
 [deps]
 AxisArrays = "39de3d68-74b9-583c-8d2d-e117c070f3a9"
+ImageBase = "c817782e-172a-44cc-b673-b171935fbb9e"
 ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 SimpleTraits = "699a6c99-e7fa-54fc-8d76-47d257e15c1d"
 
 [compat]
 AxisArrays = "0.3, 0.4"
+ImageBase = "0.1.1"
 ImageCore = "0.8.1, 0.9"
 Reexport = "0.2, 1.0"
 SimpleTraits = "0.8, 0.9"
 julia = "1"
 
 [extras]
+ColorVectorSpace = "c3611d14-8923-5661-9e6a-0046d554d3a4"
```

Comment:
ColorVectorSpace is already loaded by ImageCore 0.9. Perhaps we should just standardize on 0.9? Or is it transitionally useful to bridge the gap? (I'm fine with this if so, it's a very minor point.)

Reply:
Yes, I just planned to set all our packages on top of ImageCore 0.9 and drop compatibilities with old versions https://github.com/JuliaImages/Images.jl/issues/972.

Will do this in the next PR.